src/com/ableton/Pyenv.groovy

```diff
@@ -44,19 +44,21 @@ class Pyenv implements Serializable {
     }
 
     VirtualEnv venv = new VirtualEnv(script, randomSeed)
-    int result = venv.script.sh(
+    int result
+    List installCommands = [
+      "export PYENV_ROOT=${pyenvRoot}",
+      "export PATH=\$PYENV_ROOT/bin:\$PATH",
+      "eval \"\$(pyenv init --path)\"",
+      "eval \"\$(pyenv init -)\"",
+      "pyenv install --skip-existing ${pythonVersion}",
+      "pyenv shell ${pythonVersion}",
+      'pip install virtualenv',
+      "virtualenv ${venv.venvRootDir}",
+    ]
+    result = venv.script.sh(
```

Comment:
In de32499621186c7c7d63e2bac97f5c8bcf46e265, I don't see why `result` gets defined before `installCommands`.

Reply:
See 14fbba1.